pkgs/desktops/deepin/go-package/go-lib/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, replaceAll
+, glib
+, xorg
+, gdk-pixbuf
+, pulseaudio
+, mobile-broadband-provider-info
+, runtimeShell
+}:
+
+stdenv.mkDerivation rec {
+  pname = "go-lib";
+  version = "5.8.27";
+
+  src = fetchFromGitHub {
+    owner = "linuxdeepin";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-ZV5zWu7UvNKVcVo79/iKMhF4H09rGyDCvEL61H05lZc=";
+  };
+
+  # https://github.com/linuxdeepin/go-lib/pull/80
+  patches = [ ./0001-fix-IsDir-for-symlink.patch ];
+
+  postPatch = replaceAll "/bin/sh" "${runtimeShell}";
+
+  propagatedBuildInputs = [
```

Comment:
Can we solve this differently than polluting all depending packages with those packages?

Reply:
we can set this as dde-daemon‘s buildInputs